attribute/jsonattribute.go

```diff
@@ -5,20 +5,20 @@ import (
 	"fmt"
 
 	"github.com/getyoti/yoti-go-sdk/anchor"
-	"github.com/getyoti/yoti-go-sdk/yotiprotoattr_v3"
+	"github.com/getyoti/yoti-go-sdk/yotiprotoattr"
 )
 
 //JSONAttribute is a Yoti attribute which returns an interface as its value
 type JSONAttribute struct {
-	*yotiprotoattr_v3.Attribute // Value returns the value of a JSON attribute in the form of an interface
-	value                       interface{}
-	anchors                     []*anchor.Anchor
-	sources                     []*anchor.Anchor
-	verifiers                   []*anchor.Anchor
+	*yotiprotoattr.Attribute // Value returns the value of a JSON attribute in the form of an interface
+	value                    interface{}
+	anchors                  []*anchor.Anchor
+	sources                  []*anchor.Anchor
```

Comment:
Do you need `sources` and `verifiers` on top of `anchors`?

Reply:
I meant only pass in anchors and parse sources and verifiers from that.

Reply:
Ah yes good point, will change